docs/source/quickstarter.rst

```diff
@@ -2,17 +2,41 @@
 Quickstarter
 ============
 
-Install packages
--------------
+Data input
+----------
+
+The only input data required for *naturf* is a shapefile with building footprints and height data. There should be a field with a unique ID for each building the shapefile, and it should be projected in WGS 84 (EPSG:4326). 
+
+Clone git repository
+--------------------
+
+.. code:: bash
+
+    git clone https://github.com/IMMM-SFA/naturf.git
+
+Install dependencies
+--------------------
+In the directory where the *naturf* repository was cloned, run the following command to install all dependencies in the current environment.
+
+.. code:: bash
+
+    pip install .
+
+Edit config file
+----------------
+
+The *config.py* file in the naturf/ directory sets the default names for variables used in the *naturf* workflow. The two variables below need to be modified to reflect the ID field and the building height field of the input shapefile.
 
 .. code:: ipython3
 
-    import naturf
+    data_id_field_name
+    data_height_field_name
 
 
 Run `naturf` using hamilton
 ---------------------------
-This will run all functions required to create the output specified in `hamilton_run.py` `output_columns` variable. Currently `write_binary` and `write_index`.
+This will run all functions required to create the output specified in `hamilton_run.py` `output_columns` variable. Currently `write_binary` and `write_index`. The `path` variable should point towards the input shapefile.
```

Comment:
Do we want these italicized or inline code block? I'm not sure how to do the latter.

Reply:
_italicized_ is \_underscore\_, `inline code` is \`backticks\`. I think since these are variables having it as inline code is good!